.env.example

```diff
@@ -1 +1 @@
-REACT_APP_SERVER_URL=http://localhost:3001
```

Comment:
this changes shouldn't be here. You need to add this URL only to your local machine file called .env

Reply:
I did this for another developer while we build project. Will remove.